src/schema/v2/schema.ts

```diff
@@ -293,6 +297,8 @@ const rootFields = {
   identityVerificationsConnection,
   job,
   jobs,
+  saleAgreement: SaleAgreement,
```

Comment:
Wondering about the use case for a root-field `saleAgreement` that has to be found by sale agreement id (this is the only use-case for `saleAgreementLoader`). If it's always associated with a sale, I might expect its use to be via `saleAgreement` under a `Sale`, vs. a root-field.

Reply:
Fair question. Ultimately the client, `force` in this case, only cares about the sale agreement and the name of the associated sale. If we only exposed this via the parent Sale, we would need to add a lot of filtering support which would only apply to a small subset of sales. Felt simpler, and ultimately cleaner to expose the `SaleAgreement` directly via a schema.